### PR TITLE
Make individual_metadata_items optional to fix deprecation warning

### PR DESCRIPTION
**Pull request by rogerluan**
🔑 

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass (I ran `rspec deliver` tbh)
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary. (I couldn't find any documentation to be updated 😕 I'm assuming it'll be auto-generated from the model I updated)

### Motivation and Context

When running `deliver` I was seeing this warning:

```
Using deprecated option: '--individual_metadata_items' (Removed after the migration to the new App Store Connect API in June 2020)
```

Even though I wasn't using any `individual_metadata_items`. 

### Description

My hypothesis was that if I made this property optional, the warning would be gone.
I made it optional, ran `be fastlane generate_swift_api` and re-tested, and confirmed the warning was gone!
Throughout the process, I had a hard time figuring out what I had to call in order to re-generate the Swift APIs. So now I also added documentation for this step, for newcomers like me 🙈 

### Testing Steps

In latest release, run deliver using Swift API, you will see the warning.
When running this branch, the warning is gone (assuming you're actually not sending those parameters 😄 )

### CODE REVIEW

**Comment by rogerluan** [comment]
@googlebot I signed it!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
 - [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
 - [ ] I've updated the documentation if necessary.
+- [ ] I've run `bundle exec fastlane generate_swift_api` if necessary.
```
Comment (joshdholtz) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: You do not need to run `generate_swift_api`. This is done when new versions of fastlane are made. Can we remove this? 😊

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ Upon your first pull request to _fastlane_, the [googlebot](https://github.com/g
 
 Please be aware that we don’t accept submissions for new actions at the moment. You can find more information about that [here][submit action].
 
+### Updating Auto-Generated Swift API
+
+If your changes update any interface, documentation or behavior of _fastlane_ APIs, it's very likely these changes will require updating the Swift APIs.
+You can update the auto-generated Swift APIs by running `bundle exec fastlane generate_swift_api`. Commit the changes generated and include them in your PR.
+
```
Comment (joshdholtz) [inline CONTRIBUTING.md:34]: This can also be removed

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Thanks for catching this! Just got some requested changes about generating the Swift API. This happens when we release new version of fastlane so if you could be able to remove all of those changes that would be 💯 

Thanks! ❤️

**Comment by rogerluan** [comment]
@joshdholtz done!

Since I still feel like it lacks documentation around this subject, I moved the discussion about the documentation to a separate PR: https://github.com/fastlane/fastlane/pull/16813 - Looking forward to hearing your comments there! 

So now this PR is unblocked from that separate discussion.

Thanks 🚀

**Review by joshdholtz — APPROVED** [review]
This is perfect! Thanks for fixing ❤️  Appreciate the contribution!

**Pull request merged**